test/solium/no-arg-overlow-rule.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+let fs = require('fs');
+let dedent = require("dedent");
+let Solium = require("solium");
+
+let userConfig = JSON.parse(fs.readFileSync('.soliumrc.json', 'utf8'));
+
+describe("[RULE] arg-overflow", function() {
+
+  afterEach(function(done) {
+    Solium.reset();
+    done();
+  });
+
+  it("should ignore more than 3 arguments in the same line", function(done) {
+    let code = dedent`
+                pragma solidity ^0.4.3;
+
+
+                contract Dummy {
+                  function testFunc(uint arg1, uint arg2, uint arg3, uint arg4) public {
+                  }
+                }`,
+        errors = Solium.lint(code, userConfig);
```

Comment:
I find the `let a = expr, b = expr` syntax confusing, is there a reason to have both assignments on the same line?